### ISSUE TITLE
TypeError: Cannot read property 'document' of undefined

### ISSUE THREAD

**Issue by alexanderadam** [j0meinaster/bracket-peek#15]
Somehow I got this error in console:

```
ERR Cannot read property 'document' of undefined: TypeError: Cannot read property 'document' of undefined
    at getEnabledClosingsRegex (/a/path/.vscode/extensions/jomeinaster.bracket-peek-1.4.2/src/utils/closures.js:38:20)
    at regexLastIndexOfClosing (/a/path/.vscode/extensions/jomeinaster.bracket-peek-1.4.2/src/utils/indexOf.js:12:19)
    at /a/path/.vscode/extensions/jomeinaster.bracket-peek-1.4.2/src/extension.js:58:26
    at fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
    at D.$acceptEditorPropertiesChanged (/some/path/workbench/services/extensions/node/extensionHostProcess.js:88:46550)
    at l._doInvokeHandler (/some/path/workbench/services/extensions/node/extensionHostProcess.js:91:12836)
    at l._invokeHandler (/some/path/workbench/services/extensions/node/extensionHostProcess.js:91:12520)
    at l._receiveRequest (/some/path/workbench/services/extensions/node/extensionHostProcess.js:91:11187)
    at l._receiveOneMessage (/some/path/workbench/services/extensions/node/extensionHostProcess.js:91:9973)
    at /some/path/workbench/services/extensions/node/extensionHostProcess.js:91:8074
    at fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
    at S.fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:65:15497)
    at /some/path/workbench/services/extensions/node/extensionHostProcess.js:106:29764
    at fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
    at S.fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:65:15497)
    at t._receiveMessage (/some/path/workbench/services/extensions/node/extensionHostProcess.js:65:20755)
    at /some/path/workbench/services/extensions/node/extensionHostProcess.js:65:17641
    at fire (/some/path/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
    at acceptChunk (/some/path/workbench/services/extensions/node/extensionHostProcess.js:65:12862)
    at /some/path/workbench/services/extensions/node/extensionHostProcess.js:65:12210
    at Socket.v (/some/path/workbench/services/extensions/node/extensionHostProcess.js:106:13195)
    at Socket.emit (events.js:315:20)
    at Socket.EventEmitter.emit (domain.js:467:12)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:188:23)
```

**PS:** Thank you for the bracket-peek extension 🙏

**Comment by crazycliffy** [comment]
There was a new release of bracket-peek v1.4.3 yesterday (Mar21st).  After upgrading, I am getting many exceptions in the "Runtime Status" for Bracket Peek.

```
Cannot read property 'document' of undefined
Cannot access 'activeEditor' before initialization
```


I am using this setup

```
vscode v1.65.2
ms-vscode.cpptools v1.8.4
ms-vscode-remote.remote-ssh v0.76.1
jomeinaster.bracket-peek v1.4.3
```

When looking at the Log (Window), here is the stacktrace.

```
[2022-03-22 10:17:07.358] [renderer1] [error] Cannot read property 'document' of undefined: TypeError: Cannot read property 'document' of undefined
	at module.exports (/home/user/.vscode-server/extensions/jomeinaster.bracket-peek-1.4.3/src/utils/closest-closing.js:18:35)
	at Object.provideHover (/home/user/.vscode-server/extensions/jomeinaster.bracket-peek-1.4.3/src/extension.js:83:29)
	at a.provideHover (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:85:94289)
	at /home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:85:119819
	at Ee._withAdapter (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:85:116655)
	at Ee.$provideHover (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:85:119797)
	at n._doInvokeHandler (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:89:13812)
	at n._invokeHandler (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:89:13496)
	at n._receiveRequest (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:89:12107)
	at n._receiveOneMessage (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:89:10836)
	at /home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:89:8945
	at d.invoke (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
	at b.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:1856)
	at c.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:19017)
	at /home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:104:34267
	at d.invoke (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
	at b.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:1856)
	at c.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:19017)
	at s._receiveMessage (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:23598)
	at /home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:21132
	at d.invoke (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
	at b.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:1856)
	at b.acceptChunk (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:15848)
	at /home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:66:14978
	at d.invoke (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:145)
	at b.fire (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:58:1856)
	at l._processReadQueue (/home/user/.vscode-server/bin/xx_xx_d1/out/vs/workbench/api/node/extensionHostProcess.js:104:21272)
```

I downgraded to last years version of bracket-peek v1.4.2, and the problem went away.

**Comment by j0meinaster** [comment]
Sorry. I messed up 1.4.3.. Thanks for the report. should be fixed in 1.4.4

**Comment by crazycliffy** [comment]
> Sorry. I messed up 1.4.3.. Thanks for the report. should be fixed in 1.4.4

Verified working! thanks for posting the fix. 😀😀.  BTW with your feature, developers no longer need to copy-paste and put comments at the of an extremely long code section at the end of the bracket. this extension takes care of that.
